app/src/main/java/com/duckduckgo/app/browser/BrowserActivity.kt

```diff
@@ -613,6 +617,10 @@ class BrowserActivity : DuckDuckGoActivity(), BookmarkDialogCreationListener, We
         popupMenu.show(rootView, toolbar)
     }
 
+    private fun launchSharePageChooser(url: String) {
+        share(url, "")
```

Comment:
I don't see Chrome appear as an option here. I wonder if that works via `browse`. Not sure how easy it is but it would be good if we could cover _all_ the apps supported by `share` and `browse`.

Reply:
We may need two options "share" like you have now and "open in" for browse